widgetssdk/src/main/java/com/glia/widgets/view/VisitorCodeView.kt

```diff
@@ -100,6 +99,9 @@ class VisitorCodeView internal constructor(
         runOnUi {
             showProgressBar(true)
             showSuccess()
+            successTitle.contentDescription = context.getString(R.string.glia_visitor_code_loading)
+            successTitle.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUSED)
+            closeButton.contentDescription = context.getString(R.string.glia_chat_alert_dialog_close_content_description)
```

Comment:
Is there some specific reason why the close button content description is done programmatically? I guess it is related to the:
>For this reason I disabled contentDescription for close and glia image because those kept coming up in undesired ways.

Could be useful to have an in-code comment so that someone during refactoring does not move it back to XML